resources/js/features/games/hooks/useAllMetaRowElements.ts

```diff
@@ -0,0 +1,322 @@
+  );
+
+  const creditRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Credits -'],
+        primaryLabel: 'Credits',
+      }),
+    [allGameHubs],
+  );
+
+  const technicalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Technical -'],
+        primaryLabel: 'Technical',
+      }),
+    [allGameHubs],
+  );
+
+  const protagonistRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Protagonist -'],
+        primaryLabel: 'Protagonist',
+      }),
+    [allGameHubs],
+  );
+
+  const settingRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Setting -'],
+        primaryLabel: 'Setting',
+      }),
+    [allGameHubs],
+  );
+
+  const regionalRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Regional -'],
+        primaryLabel: 'Regional',
+      }),
+    [allGameHubs],
+  );
+
+  const raFeatureRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Meta -', 'RANews -', 'Custom Awards -'],
+        hubTitleExcludes: ['Meta - Language'],
+        primaryLabel: 'Meta',
+        altLabels: ['RANews', 'Custom Awards'],
+      }),
+    [allGameHubs],
+  );
+
+  const formatRowElements = useMemo(
+    () =>
+      buildMetaRowElements({
+        hubs: allGameHubs,
+        hubTitleIncludes: ['Format -'],
+        primaryLabel: 'Format',
+      }),
+    [allGameHubs],
+  );
+
+  // Calculate used hub IDs from all other categories (excluding misc).
+  const usedHubIdsFromOtherCategories = useMemo(() => {
+    const allRows = [
+      ...creditRowElements,
+      ...developerRowElements,
+      ...featureRowElements,
+      ...formatRowElements,
+      ...genreRowElements,
+      ...hackOfRowElements,
+      ...languageRowElements,
+      ...perspectiveRowElements,
+      ...protagonistRowElements,
+      ...publisherRowElements,
+      ...raFeatureRowElements,
+      ...regionalRowElements,
+      ...settingRowElements,
+      ...technicalRowElements,
+      ...themeRowElements,
+    ];
+
+    return new Set(allRows.map((row) => row.hubId).filter((id): id is number => Boolean(id)));
+  }, [
+    creditRowElements,
+    developerRowElements,
+    featureRowElements,
+    formatRowElements,
+    genreRowElements,
+    hackOfRowElements,
+    languageRowElements,
+    perspectiveRowElements,
+    protagonistRowElements,
+    publisherRowElements,
+    raFeatureRowElements,
+    regionalRowElements,
+    settingRowElements,
+    technicalRowElements,
+    themeRowElements,
+  ]);
+
+  const miscRowElements = useMemo(
+    () =>
+      buildMiscRowElements(allGameHubs, usedHubIdsFromOtherCategories, {
+        keepPrefixFor: ['Fangames'],
```

Comment:
`Rollout Sets - Mega Duck` and `The Unwanted - Past Games` are appearing under both Misc and Events.
http://localhost:64000/game2/20221
![image](https://github.com/user-attachments/assets/2a0ac10e-3b94-4997-abc5-eaeb69b34f12)

If they should be under Misc, they should keep their prefix.

And `Freely Available in Forum Topic` could be an "Achievement Extra"?

Reply:
Fixed in latest so event hubs will no longer appear in Misc. They should appear in Events only.

I'm not sure "Freely Available in Forum Topic" is a good fit for achievement extra. To me, this is more _game-specific_ data -- an indicator that the game is freeware.

Reply:
I guess "Freely Available" is fine where it is. I was just thrown off by the "Forum Topic" subclause and assumed it should go into the "RA Features" bucket, which has been renamed to "Achievement Extras".

Confirmed doubly linked events are now only in Events section.